ts-tests/scripts/clean-up.sh

```diff
@@ -1,19 +1,20 @@
 #!/bin/sh
 
-TMP_DIR="$1"
+TMP_DIR="/tmp"
+TMP_DIR="${1:-$TMP_DIR}"
+basedir=$(dirname "$0")
 
-echo "killing litentry-token-server ..."
-killall litentry-token-server
-echo "killing polkadot ..."
-killall polkadot
-echo "killing litentry-collator ..."
-killall litentry-collator
+. $basedir/constants.sh $TMP_DIR
 
-echo
+echo "Stop token server ..."
+[ -f $TOKEN_SERVER_PIDFILE ] && kill -9 $(cat $TOKEN_SERVER_PIDFILE)
```

Comment:
I think we still need `shell: bash{0} `in the workflow file when running clean-up.sh, because otherwise `set -eo pipefail` is still turned on by Github here, imagine what if killing token-server somehow fails, we'd probably hope to kill other processes..

Reply:
Yeah. I updated it